### PR TITLE
Add support for specifying database name for postgres wallet

### PR DESCRIPTION
**Pull request by joevolleyball**
MultiWalletSingleTable Postgres wallet has default DB name 'Wallets'. This allows the name to be configured during wallet creation, open and delete as well as when init'ing the storage type.

This replaces my previous PR https://github.com/hyperledger/indy-sdk/pull/1979

Signed-off-by: Aaron Reed <aaronr@us.ibm.com>

### CODE REVIEW

File: experimental/plugins/postgres_storage/src/postgres_storage.rs [hunk 1]
```diff
@@ -617,42 +633,81 @@ impl WalletStrategy for DatabasePerWalletStrategy {
...
+        None => _WALLETS_DB,
+    };
+
+    return name;
+}
+
 impl WalletStrategy for MultiWalletSingleTableStrategy {
     // initialize storage based on wallet storage strategy
     fn init_storage(&self, config: &PostgresConfig, credentials: &PostgresCredentials) -> Result<(), WalletStorageError> {
+        match env_logger::try_init() {
```
Comment (Artemkaaas) [inline experimental/plugins/postgres_storage/src/postgres_storage.rs]: Why did you add logger initialization `init_storage` functions?  The logger will already be initialized on library loading here:
 https://github.com/hyperledger/indy-sdk/blob/5d3587a183399a14da680bb4e7e4c8540d1f6dea/experimental/plugins/postgres_storage/src/lib.rs#L59
Either Libindy callbacks or default env_logger will be set.

File: experimental/plugins/postgres_storage/src/postgres_storage.rs [hunk 2]
```diff
@@ -617,42 +633,81 @@ impl WalletStrategy for DatabasePerWalletStrategy {
...
+        let wallet_db_name: &str = get_multi_database_name(config);
+        debug!("wallet_db_name: {:?}", wallet_db_name);
         let url_base = PostgresStorageType::_admin_postgres_url(&config, &credentials);
-        let url = PostgresStorageType::_postgres_url(_WALLETS_DB, &config, &credentials);
-
-        let conn = postgres::Connection::connect(&url_base[..], postgres::TlsMode::None)?;
+        let url = PostgresStorageType::_postgres_url(wallet_db_name, &config, &credentials);
+        debug!("postgres_url: {:?}", url);
+        debug!("connecting to postgres, url_base: {:?}", url_base);
+        let conn = postgres::Connection::connect(&url_base[..], config.tls())?;
```
Comment (Artemkaaas) [inline experimental/plugins/postgres_storage/src/postgres_storage.rs]: Why do you use here `config.tls()` as the last parameter but on the line 699 use `postgres::TlsMode::None` ? 
Is it intentional?  The same thing for `create_wallet`, `delete_wallet`

File: experimental/plugins/postgres_storage/src/postgres_storage.rs (hunk 1 above)
Comment (joevolleyball) [reply experimental/plugins/postgres_storage/src/postgres_storage.rs]: removed

File: experimental/plugins/postgres_storage/src/postgres_storage.rs (hunk 2 above)
Comment (joevolleyball) [reply experimental/plugins/postgres_storage/src/postgres_storage.rs]: changed them all to config.tls() to be consistent

**Comment by joevolleyball** [comment]
Messed up my fork with unsigned commits.  Again.  Ugh.  Will open new PR
